gamemodes/terrortown/gamemode/client/vgui/cl_sb_row.lua

```diff
@@ -281,7 +281,7 @@ end
 function GM:TTTScoreboardRowColorForPlayer(ply)
 	local col = color_trans
 
-	if IsValid(ply) and ply.GetRoleColor and ply:GetRoleColor() and ply:GetSubRole() and ply:GetSubRole() ~= ROLE_INNOCENT then
+	if IsValid(ply) and ply.GetRoleColor and ply:GetRoleColor() and ply:GetSubRole() and ply:GetSubRole() ~= ROLE_NONE then
```

Comment:
This means every innocent player would see themselves colored (green) in the scoreboard, right?

Reply:
innocent players are green now, yes. But I think this is alright.

Reply:
Yea. Green players in the scoreboard are confirmed innocents now - like the other roles are doing :D